fe/fe-core/src/test/java/com/starrocks/sql/plan/PlanFragmentWithCostTest.java

```diff
@@ -890,13 +876,10 @@ public void testNotPushDownRuntimeFilterAcrossCTE() throws Exception {
                 "    select * from cte union all\n" +
                 "    select cte.v4, cte.v5, cte.v6 from t2 join cte on cte.v4 = t2.v7 and t2.v8 < 10) as t\n";
         String plan = getVerboseExplain(sql);
-        Assert.assertTrue(plan.contains("  0:OlapScanNode\n" +
+        Assert.assertTrue(plan, plan.contains("  0:OlapScanNode\n" +
```

Comment:
may be Assert.assertTrue(plan.contains(...)) ?

Reply:
The first parameter is just used to print a error message for debugging.